hack/lib/release-images.sh

```diff
@@ -83,21 +83,46 @@ function build_multi_arch_binaries() {
 
 function build_docker_image() {
     for arch in ${target_arch[@]}; do
-        for binary in "${bin_targets_without_servant[@]}"; do
+        for binary in "${bin_targets_process_servant[@]}"; do
            local binary_name=$(get_output_name $binary)
            local binary_path=${YURT_LOCAL_BIN_DIR}/${SUPPORTED_OS}/${arch}/${binary_name}
            if [ -f ${binary_path} ]; then
                local docker_build_path=${DOCKER_BUILD_BASE_IDR}/${SUPPORTED_OS}/${arch}
                local docker_file_path=${docker_build_path}/Dockerfile.${binary_name}-${arch}
                mkdir -p ${docker_build_path}
 
-               local yurt_component_image="${REPO}/${binary_name}:${TAG}-${arch}"
-               local base_image="k8s.gcr.io/debian-iptables-${arch}:v11.0.2"
-               cat <<EOF > "${docker_file_path}"
+               local yurt_component_image
+               local base_image
+               if [[ ${binary} =~ yurtctl ]]
+               then
+                 yurt_component_image=$REPO/yurtctl-servant:$TAG-$arch
+                 case $arch in
+                  amd64)
+                      base_image="amd64/alpine:3.9"
+                      ;;
+                  arm64)
+                      base_image="arm64v8/alpine:3.9"
+                      ;;
+                  arm)
+                      base_image="arm32v7/alpine:3.9"
+                      ;;
+                  *)
+                      echo unknown arch $arch
+                      exit 1
+                 esac
+                 cat << EOF > $docker_file_path
+FROM ${base_image}
+ADD ${binary_name} /usr/local/bin/yurtctl
+EOF
+               else
+                 yurt_component_image="${REPO}/${binary_name}:${TAG}-${arch}"  # openyurt/${binary_name}:latest-${arch}
+                 base_image="k8s.gcr.io/debian-iptables-${arch}:v11.0.2"
+                 cat <<EOF > "${docker_file_path}"
```

Comment:
nit: This change looks like a generic refactoring, which can be submitted in an independent PR.

Reply:
This change is related to the changes in the pkg/yurtctl/constants/constants.go, maybe it's better in a pr?